source/config/Config.cpp

```diff
@@ -1763,7 +1761,10 @@ bool Config::ExportDefaultSetting(const string &file)
     "%s": {
         "%s": "DEBUG",
         "%s": "FILE",
-        "%s": "/var/log/aws-iot-device-client/aws-iot-device-client.log"
+        "%s": "/var/log/aws-iot-device-client/aws-iot-device-client.log",
+        "%s": false,
+        "%s": "TRACE",
+        "%s": "/var/log/aws-iot-device-client/sdk.log"
```

Comment:
We can use constants storing default values over here and above.

Reply:
Done.